sauce/features/budget/display-goal-amount/index.js

```diff
@@ -0,0 +1,88 @@
+import Feature from 'core/feature';
+import * as toolkitHelper from 'helpers/toolkit';
+
+export default class DisplayTargetGoalAmount extends Feature {
+  constructor() {
+    super();
+  }
```

Comment:
You actually don't need to specify a constructor if it doesn't do anything other than call `super();`. That's ok, I'll go ahead and fix it up, just wanted to let you know.